packages/api/src/auth/decoders/index.ts

```diff
@@ -21,7 +21,7 @@ interface Req {
   context: LambdaContext
 }
 
-type Decoded = null | string | Record<string, unknown>
+export type Decoded = null | Record<string, unknown>
```

Comment:
It can be a string unfortunately, for example with fire base, if no decoder is built in.

Reply:
What do you mean "no decoder is built in"
Looking here https://github.com/firebase/firebase-admin-node/blob/b2a28aee5cac172ed183bc47f71a55b31f79696a/src/auth/base-auth.ts
it says
```
   * @returns A promise fulfilled with the
   *   token's decoded claims if the ID token is valid; otherwise, a rejected
   *   promise.
```

Reply:
Only other mention I find of `verifyIdToken` is this, which is basically the same

https://github.com/firebase/firebase-admin-node/blob/94dd7c3efb9ff00b0462cf772b803d6abecc2dcc/src/auth/tenant-manager.ts#L88-L100

Reply:
The return type for firebase in our code:
```
export const firebase = async (
  token: string
): Promise<null | Record<string, unknown>> => {
```

Reply:
Hmmm I think maybe that was the old way of doing firebase - so the behaviour now is that we consistently return null or an object.

Worth double checking this (I just did a quick scan of all the decoders) - but will resolve for now!